packages/tools/generator-component/generators/app/templates/src/index.js

```diff
@@ -33,9 +32,7 @@ if (GlobalVue) {
 }
 
 // To allow use as module (npm/webpack/etc.) export component
-export default <%= name.component %>;
-
-<% } else { %>
+export default <%= name.component %>;<% } else { %>
```

Comment:
Files generated from this template had something like 3 blank lines at the end of the file